cs_scanimage.py

```diff
@@ -118,20 +106,36 @@ def get_scanreport(self, token):
             if resp.status_code != 200:
                 print("report not generated yet, retrying ... ")
             else:
-                return resp.json()
+                return ScanReport(resp.json())
         print("retries exhausted")
         raise APIError('GET ' + get_url + ' {}'.format(resp.status_code))
 
+
+class ScanReport(dict):
+    """Summary Report of the Image Scap"""
```

Comment:
@isimluk I assume that you mean `Scan` instead of `Scap`?

Reply:
muscle memory, you know...

Reply:
lol